### ISSUE TITLE
PostCSS and/or Tailwind's `@apply` support?

### ISSUE THREAD

**Issue by NullVoxPopuli** [tree-sitter/tree-sitter-css#17]
Hello!

I'm using Tailwind, which supports this syntax:

```css
.selector {
  @apply variable-a meta-variable-b lots-of-combined-properties-c;
}
```
but with the current parser, I get a squiggly on everything after `variable-a`

Is PostCSS / varying grammars in scope for tree-sitter-css?

**Comment by harrisoncramer** [comment]
I'm encountering this same problem. Treesitter fails to parse anything beyond the first value after the `@apply` directive.

![Screenshot from 2022-01-12 17-16-15](https://user-images.githubusercontent.com/32515581/149231815-95033765-5eb3-47b7-a020-4dd9aa345f4f.png)

**Comment by megalithic** [comment]
@harrisoncramer @NullVoxPopuli either of you discover fixes or build your own queries for this?

**Comment by luckasRanarison** [comment]
I can't believe that this issue has been open for 2 years

**Comment by amaanq** [comment]
There's some basic postcss support now, and I released 0.20.0
